deployment/sandbox-v2/README.md

```diff
@@ -68,16 +68,16 @@ ssl:
   certificate: <certificate dir>
   certificate_key: <private key path> 
 ```
-* Set **private ip** address of `mzworker0.sb` and `dmzworker0.sb` in `group_vars/all.yml`:
+* Set **private ip** address of `mzworker0` and `dmzworker0` in `group_vars/all.yml`:
```

Comment:
Not entirely sure. It doesn't seem to be essential for AWS deployment either judging by a quick look at some examples using Terraform and AWS.

Reply:
The clues I have a 'service broker', mainly because of the prevalence of 'sb' in [this](https://learnk8s.io/cloud-resources-kubernetes) blog.

Reply:
I would guess that the naming convention of adding '.sb' was internal rather than than essential. Perhaps our deployment would have still worked if we changed all instances to include '.sb' rather than removed it.

Reply:
Yeah, I'm (pretty) sure it would, and we'd have avoided the warnings about groups and hosts having the same names.